components/Article/Article.styled.js

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
+import Link from "next/link";
+import { css } from "styled-components";
+
+const StyledArticle = styled.article`
+  margin: 0 25px 25px 25px;
+
+  position: relative;
+  background-color: white;
+  border: none;
+  border-radius: var(--border-radius);
+`;
+
+const StyledDetailsLink = styled(Link)`
+  height: 35px;
+  position: absolute;
+  right: 10px;
+  bottom: 10px;
+  border-style: none;
+  color: var(--tertiary-color);
+`;
+
+const StyledArticleImageContainer = styled.div`
+  position: relative;
+  width: 100%;
+  height: 30vh;
+  background-color: var(--article-card);
+  border-radius: var(--border-radius) var(--border-radius) 0 0;
+`;
+
+const StyledList = styled.ul`
+  list-style: none;
+  margin: 0;
+  padding: 0 0 15px 20px;
+`;
+
+const StyledListItem = styled.li`
+  list-style: none;
+  font-size: 13px;
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  ${({ variant }) =>
+    variant === "hide" &&
+    css`
+      display: none;
+    `};
+`;
```

Comment:
- The `list-style` declaration for the `<li>` has no effect.  `list-style` should always be apply it to the list `<ol>` or `<li>`. But don't worry, because you already did.
- Regarding the Accessibility, I highly recommend to use a relative unit for the font-size. Her is a Link to a super cool [article of Josh W. Comeau ](https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/) - Of course, who else? 👑


```suggestion
const StyledListItem = styled.li`
  font-size: 0.8rem;
  display: grid;
  grid-template-columns: 1fr 2fr;
  ${({ variant }) =>
    variant === "hide" &&
    css`
      display: none;
    `};
`;
```